checkin_collection.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A collection of check-in",
+  "type": "object",
+  "properties": {
+    "check_in_items": {
+      "description": "List of check-in",
+      "id": "check_in_items",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "po_line_id": {
+            "description": "The id of the checkin po_line",
+            "type": "string",
+            "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+          },
+          "checked_in": {
+            "description": "The number of items to check-in",
+            "type": "integer"
+          },
+          "pieces": {
+            "description": "A collection of piece records",
+            "type": "array",
+            "id": "pieces",
+            "piece": {
```

Comment:
Please change `piece` to `items` because jaxrs-code-generator will not be able to create proper model but `Object` will be used instead i.e. the result is
```java
    /**
     * A collection of piece records
     * 
     */
    @JsonProperty("pieces")
    @JsonPropertyDescription("A collection of piece records")
    @Valid
    private List<Object> pieces = new ArrayList<Object>();
```